Gemfile

```diff
@@ -1,17 +1,18 @@
 source 'https://rubygems.org'
 
 gem 'berkshelf', '~> 4.3'
-gem 'chef', '~> 12.4'
+gem 'chef', '~> 12.4', '<= 12.13.37'
```

Comment:
Alright then, I'll add that reason. My issue is fixed in https://github.com/chef-cookbooks/compat_resource/pull/108